pkg/codegen/testing/test/helpers.go

```diff
@@ -374,3 +374,6 @@ const (
 	RandomSchema      SchemaVersion = "4.11.2"
 	EksSchema         SchemaVersion = "0.37.1"
 )
+
+// PulumiDotnetSDKVersion is the version of the Pulumi .NET SDK to use in program-gen tests
+const PulumiDotnetSDKVersion = "3.59.0"
```

Comment:
Added this so that program-gen can install and test generated programs against a specific version of .NET Pulumi SDK. In this case, latest version that contains the `Output.JsonSerialize` function. In case of `pulumi convert`, it already emits `3.*` so latest is resolved there as well